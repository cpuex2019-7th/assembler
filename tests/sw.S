    li x1, 0x7eadbeef
    sw x1, sp, 34
    lw a0, sp, 34
    
