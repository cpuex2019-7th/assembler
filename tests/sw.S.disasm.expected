lui ra, 518876
addi ra, ra, -273
sw ra, sp, 34
lw a0, sp, 34
